packages/public-api/src/app.ts

```diff
@@ -1,20 +1,22 @@
 import express from 'express';
-import {
-  endpointMissingMiddleware,
-  errorHandlerMiddleware,
-  parseJson,
-  cors,
-} from './middleware/common';
-import healthCheckRouter from './healthcheck/router';
-import websiteRouter from './website/router';
+import * as trpcExpress from '@trpc/server/adapters/express';
+import { endpointMissingMiddleware, errorHandlerMiddleware, cors } from './middleware/common';
+import { appRouter, createContext } from './router';
 
 export function createApp() {
   const app = express();
 
-  app.use(cors);
-  app.use(parseJson);
-  app.use(healthCheckRouter);
-  app.use(websiteRouter);
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
Can we use `!isTest()` shortcut here?

Reply:
The reason was that I didn't find a way to support cors requests in supertest when I changed router tests. Googled a bit now and replaced this condition with `.set('Origin', 'http://localhost:3000')` in the tests.